sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -604,6 +604,12 @@ pi_result _pi_context::finalize() {
     if (ZeCommandList)
       ZE_CALL(zeCommandListDestroy, (ZeCommandList));
   }
+
```

Comment:
Just curious, why is this required?

Reply:
Otherwise the global count always grows and soon reaches the limit of max created command-lists